ingestion/functions/parsing/peru/peru.py

```diff
@@ -19,7 +19,6 @@
 def convert_date(raw_date: str):
     """
     Convert raw date field into a value interpretable by the dataserver.
-
```

Comment:
please keep this PR about columbia, remove Peru changes from it.

Reply:
these should be removed now